vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/core/AddDataFromRdfFile.java

```diff
@@ -44,17 +44,21 @@
 import org.semanticweb.vlog4j.examples.ExamplesUtils;
 
 /**
+ * This example shows how facts can be imported from {@code .nt.gz} files.
```

Comment:
Since the example classes are most probably the first lines of code a user would read, I suggest mentioning the N-Triples format explicitly at the end of the Javadoc, with a link to the online W3C specification (like in RDFFileDataSource). Indeed, it is inconsistent with the ".csv" example javadoc, but we believe it is important to mention this format here, besides just the ".nt" extension.